### DISCUSSION TITLE
Plugin coexistence

### DISCUSSION THREAD

**Discussion by gmartinerm** [potmat/homebridge-google-nest-sdm#171] [General]
Hi there,
I am new to homebridge.  I have added the following plugin just for my protects

(https://github.com/chrisjshull/homebridge-nest

But now I want to install this plugin and I am getting kind of confusing following the homebridge-google-nest-sdm directions.  I am not sure how much of the previous setup I can reuse for this plugin. I am ignorant about this stuff but I am a fast learner.

**Comment by potmat** [comment]
You probably can't really use any of the previous setup.  You will need to add a new configuration for this plugin.
